Coordinate.java

```diff
@@ -1,14 +1,3 @@
-public class Coordinate {
-    private final int x;
-    private final int y;
-    public Coordinate(int x, int y) {
-        this.x = x;
-        this.y = y;
-    }
-    public int getX() {
-        return x;
-    }
-    public int getY() {
-        return y;
-    }
+public record Coordinate(int xOrdinate, int yOrdinate) {
+    
```

Comment:
Just for our sake, and learning experience, do u mind adding a comment why are u using record ;)